### ISSUE TITLE
FetchError: invalid json response

### ISSUE THREAD

**Issue by brunocalado** [BlitzKraig/fvtt-DancingLights#4]
I'm getting this error on update.

```
foundry.js:24181 FetchError: invalid json response body at https://github.com/BlitzKraig/fvtt-DancingLights/releases/download/latest/module.json reason: Unexpected token N in JSON at position 0
    at C:\Program Files\FoundryVTT\resources\app\node_modules\node-fetch\lib\index.js:272:32
    at async checkPackage (C:\Program Files\FoundryVTT\resources\app\dist\packages\views.js:1:2689)
    at async Object.installPackage (C:\Program Files\FoundryVTT\resources\app\dist\packages\views.js:1:3498)
    at async setupPost (C:\Program Files\FoundryVTT\resources\app\dist\views.js:1:7169)

```
![image](https://user-images.githubusercontent.com/4999783/86798062-c4c96c80-c046-11ea-89c9-ffb25eb2b32f.png)

**Comment by BlitzKraig** [comment]
Please try v 1.4.2, this was a 1.4.0 issue I was having, sorry!

**Comment by tcrory** [comment]
I'm seeing the same error in Foundry VTT when I try to update to the latest.

**Comment by tcrory** [comment]
So if you click the Update button in Foundry, you'll get an error when you try to upgrade. If you uninstall and reinstall, it will actually install the latest without any issues.

Maybe Foundry is trying to use the manifest and download fields in the existing module.json file (mine was 1.3.2) and the changes there are causing the update to try to grab the wrong file? Regardless, uninstalling and reinstalling is a valid workaround that may only be needed to get over the one release with a bug in the module.json file.

**Comment by tcrory** [comment]
Also, I can confirm that uninstalling/reinstalling did not screw up any of my scenes that were using Dancing Lights. It may be important that I did not refresh or attempt to load any games in between the uninstall/reinstall.

**Comment by BlitzKraig** [comment]
Thanks for the info tcrory.
Yeah, Foundry seems to be a bit funky with the module.json, not entirely sure what's going on.

I'll need to keep an eye on that, as each update will now have a versioned `manifest` and `download` link, so a bit worried updates are just going to re-download the same version in the future...

I'm going to keep this ticket open to remind me to check that on the next update.
Cheers

**Comment by BlitzKraig** [comment]
So I solicited some advice, and have updated the module.json to point it back at the master version, due to the way Foundry handles that file.
Users on pre 1.4.2 who update to 1.4.3+ shouldn't notice any issues now.
If you were on 1.4.2, you will now need to uninstall and reinstall the module again.

I have updated the 1.4.2 release to display a message telling users this.
In theory, when they try to update to 1.4.3 Foundry will pull the newer version of 1.4.2 down, displaying the message in game.

Apologies again for any inconvenience, this has been a bit of a tough one!
